packages/edit-site/src/components/editor/index.js

```diff
@@ -94,14 +94,15 @@ function Editor( { settings: _settings } ) {
 		[]
 	);
 
-	const blockContext = useMemo( () => {
-		if ( ! settings.page.context.queryContext )
-			return settings.page.context;
-
-		return {
+	// Set default query for misplaced Query Loop blocks, and
```

Comment:
What is a "misplaced" query loop block?

Reply:
Placed in a document without a query context.

This can also sometimes happen when switching between templates very quickly, and the rerender lags behind the context switch for some reason.